apps/ui/src/components/Modal/VoteReason.vue

```diff
@@ -16,6 +17,9 @@ defineEmits<{
     <template #header>
       <h3 v-text="'Reason'" />
     </template>
-    <div class="p-4 whitespace-pre-line text-skin-link" v-text="vote?.reason" />
+    <div
+      class="p-4 whitespace-pre-line text-skin-link"
+      v-html="autoLinkText(vote?.reason || '')"
+    />
```

Comment:
Unlike proposal page, links are not observable on reason modal and space overview page

Proposal page: 
<img width="631" alt="Untitled 6" src="https://github.com/user-attachments/assets/a96ab66e-dcd6-4bd5-a423-13e83de373cd">

Reason modal:
<img width="440" alt="Untitled 5" src="https://github.com/user-attachments/assets/6b142c59-cffc-410a-9c54-e2c990f3bef0">

Overview page
<img width="629" alt="Untitled 7" src="https://github.com/user-attachments/assets/0aada929-078d-49fe-8ae2-4efd2e719a4b">

Should we make them similar?

Reply:
What do mean "observable" ? you mean styling like the underline ?

Reply:
I agree, it's hard to recognize links with current style as they look like just text.

Reply:
Will need @bonustrack on how to style these

Reply:
Can we style links the same as in Proposal body?

Reply:
We could add `.setAttr('class', 'underline')` to 

https://github.com/snapshot-labs/sx-monorepo/blob/93739c8cc1a8e16cda179d18e3dc19ba4f9ac6d9/apps/ui/src/helpers/utils.ts#L508

But this will also enable this style on all auto linked text: 

- Space profile
- User profile
- Space user profile

Or do we enable only on this modal ?

Reply:
We can do this, similar to how we handle proposal body:
```diff
diff --git a/apps/ui/src/components/Modal/VoteReason.vue b/apps/ui/src/components/Modal/VoteReason.vue
index 193d61ea..e26598f2 100644
--- a/apps/ui/src/components/Modal/VoteReason.vue
+++ b/apps/ui/src/components/Modal/VoteReason.vue
@@ -18,8 +18,14 @@ defineEmits<{
       <h3 v-text="'Reason'" />
     </template>
     <div
-      class="p-4 whitespace-pre-line text-skin-link"
+      class="vote-reason p-4 whitespace-pre-line text-skin-link"
       v-html="autoLinkText(vote?.reason || '')"
     />
   </UiModal>
 </template>
+
+<style class="scss">
+.vote-reason a {
+  text-decoration: underline;
+}
+</style>
```

<img width="519" alt="image" src="https://github.com/user-attachments/assets/526b5055-2d2b-46b7-9053-186de5e87bf3">

Reply:
This style looks good to me

> We can do this, similar to how we handle proposal body:
> 
> ```diff
> diff --git a/apps/ui/src/components/Modal/VoteReason.vue b/apps/ui/src/components/Modal/VoteReason.vue
> index 193d61ea..ecc74f62 100644
> --- a/apps/ui/src/components/Modal/VoteReason.vue
> +++ b/apps/ui/src/components/Modal/VoteReason.vue
> @@ -18,8 +18,14 @@ defineEmits<{
>        <h3 v-text="'Reason'" />
>      </template>
>      <div
> -      class="p-4 whitespace-pre-line text-skin-link"
> +      class="vote-reason p-4 whitespace-pre-line text-skin-link"
>        v-html="autoLinkText(vote?.reason || '')"
>      />
>    </UiModal>
>  </template>
> +
> +<style class="scss" scoped>
> +.vote-reason:deep() a {
> +  text-decoration: underline;
> +}
> +</style>
> ```
> 
> <img alt="image" width="519" src="https://private-user-images.githubusercontent.com/1968722/353466042-526b5055-2d2b-46b7-9053-186de5e87bf3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIzNzQ5MjQsIm5iZiI6MTcyMjM3NDYyNCwicGF0aCI6Ii8xOTY4NzIyLzM1MzQ2NjA0Mi01MjZiNTA1NS0yZDJiLTQ2YjctOTA1My0xODZkZTVlODdiZjMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDczMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MzBUMjEyMzQ0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NGNiNmQ3MDRlYTZkNTkzYmFlZWQ5MTkzYTI5MjFkY2QxYWU2NGZiZmY4ODQyZTc3YTlmOWRjYzgxNjlmMGEzMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.p-mJSYnDy6DGm7NSXo_WVt55KHwTejU-HdpeanqM2-I">